CHANGELOG.md

```diff
@@ -70,6 +74,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 #### global
 - Add reference_path to a handful of classes
 
+#### curves.py
+- Ellipse2D: point_distance_1 and tangent_points
+
```

Comment:
```suggestion

```

Reply:
Remove?

Reply:
Do I need to write the changes in v0.17 ou v0.18 ?